plugins/catalog-backend/src/service/NextRouter.ts

```diff
@@ -119,7 +119,10 @@ export async function createNextRouter(
       })
       .delete('/entities/by-uid/:uid', async (req, res) => {
```

Comment:
Is this typically called from the frontend? If so, maybe worth double-checking as part of this PR that the token is being supplied.

Reply:
In the refresh endpoint PR, we decided to pass possibly undefined tokens through to the permission api call and let it handle the case. Happy to change this and start checking for presence of tokens if we think that would be useful, but in that case we should do it consistently over the endpoints and should probably be done in a separate PR.